ydb/docs/en/core/yql/reference/udf/list/knn.md

```diff
@@ -66,6 +66,20 @@ Knn::ToBinaryStringBit(List<Int8>{Flags:AutoMap})->Tagged<String, "BitVector">
 Knn::FloatFromBinaryString(String{Flags:AutoMap})->List<Float>?
 ```
 
+#### Convert format {#functions-convert-format}
+
+Conversion functions for vector data convert an array of elements into a byte string with the following format:
+
+- **Main part** — a contiguous array of elements ([knn-serializer.h](https://github.com/ydb-platform/ydb/blob/0b506f56e399e0b4e6a6a4267799da68a3164bf7/ydb/library/yql/udfs/common/knn/knn-serializer.h#L19))
+- **Type** — 1 byte at the end of the string that specifies the data type ([knn-defines.h](https://github.com/ydb-platform/ydb/blob/24026648dd7463d58e1470aa8981b17677116e7c/ydb/library/yql/udfs/common/knn/knn-defines.h#L5)):  
+  `1` — `Float` (4 bytes per element),  
+  `2` — `Uint8` (1 bytes per element),  
+  `3` — `Int8` (1 bytes per element),  
+  `10` — `Bit` (1 bytes per element)
```

Comment:
<img width="439" height="55" alt="Screenshot 2568-08-01 at 18 41 21" src="https://github.com/user-attachments/assets/2bb6f2c4-a29e-426a-8bec-638c191fd30f" />

The header file linked above says it's 1 bit per element instead of 1 byte. Which option is correct?

Reply:
The Russian version below also says 1 bit per element.